whats-on/terraform/services.tf

```diff
@@ -0,0 +1,34 @@
+module "whats_on" {
+  source             = "git::https://github.com/wellcometrust/terraform.git//services?ref=v1.0.4"
+  name               = "whats-on"
```

Comment:
Any chance we could use `_`?
It just makes it way more consistent with any of the other infra stuff, where dashes can be annoying.

Maybe in the folder name too?